.bingo/crd-diff.mod

```diff
@@ -0,0 +1,5 @@
+module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
+
+go 1.22.5
```

Comment:
It seems to be failing in the api-diff to build

Reply:
See: https://github.com/operator-framework/operator-controller/actions/runs/11808262605/job/32896538347?pr=1449

Reply:
Yeah, this looks like a failure that got masked by our breaking API changes. I'm planning to investigate in a bit

Reply:
https://github.com/operator-framework/operator-controller/pull/1468

Reply:
This is in now, if you rebase, it should pick up the fix for go-verdiff

Reply:
Looks to have passed now. Thanks!